### DISCUSSION TITLE
Polygon intersection onHover event

### DISCUSSION THREAD

**Discussion by markuplab** [visgl/deck.gl#8519] [Q&A]
Good afternoon, I am writing to seek your assistance with intersecting polygons. 

I have prepared a simple example to demonstrate my issue: https://stackblitz.com/edit/js-puy4b5?file=index.js. 

In this example, I would like to know how to make the "onHover" events "fall through" to the underlying polygon when the color of the overlaying polygon is transparent. 

As you can see, the topmost layer always captures the event, regardless of whether it is transparent or not. 

However, there is also a blue polygon below that the "onHover" events do not reach. 

Could there be a way to solve this issue? 
Thank you for your attention.

**Comment by Pessimistress** [comment]
Call [deck.pickMultipleObjects](https://deck.gl/docs/api-reference/core/deck#pickmultipleobjects) with `depth`.
